prime-router/src/main/kotlin/Report.kt

```diff
@@ -76,7 +76,8 @@ data class ReportStreamFilterResult(
     val filterArgs: List<String>,
     val filteredTrackingElement: String,
     val filteredIndex: Int,
-    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION
+    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION,
+    val filterType: String? = ""
```

Comment:
Maybe make `filterType` required and non-null? It looks like it should always be set.